packages/react/src/models/auth-context.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {UIAuthConfig, MeAPIResponse} from '@asgardeo/js-ui-core';
+
+interface AuthContext {
+  accessToken: string;
+  config: UIAuthConfig;
+  customizationOptions?: any;
```

Comment:
Don't use `any`. Check in other places as well

Reply:
Will address this in the next PR.

Reply:
addressed with 773fa4456ecaad5cf38ace36da321a7b85b63ce4